typescript-stl/typescript-stl.d.ts

```diff
@@ -1,5 +1,5 @@
-// Type definitions for TypeScript-STL v1.0.0-rc.2
-// Project: https://github.com/samchon/stl
+// Type definitions for TypeScript-STL v1.0.0-rc.3
+// Project: https://github.com/samchon/typescript-stl
 // Definitions by: Jeongho Nam <http://samchon.org>
```

Comment:
can you update this to use your github account, this way the dt-bot pings you if any one touches this file.